app/src/main/java/net/squanchy/eventdetails/EventDetailsActivity.java

```diff
@@ -19,5 +35,27 @@ protected void onCreate(Bundle savedInstanceState) {
 
         Toolbar toolbar = (Toolbar) findViewById(R.id.toolbar);
         setSupportActionBar(toolbar);
+
+        EventDetailsComponent component = EventDetailsInjector.obtain(this);
+        service = component.service();
+    }
+
+    @Override
+    protected void onStart() {
+        super.onStart();
+
+        Intent intent = getIntent();
+        int dayId = intent.getIntExtra(EXTRA_DAY, -1);
+        int eventId = intent.getIntExtra(EXTRA_EVENT_ID, -1);
+        subscription = service.event(dayId, eventId)
+                .observeOn(AndroidSchedulers.mainThread())
+                .subscribe(event -> Toast.makeText(this, "Event: " + event, Toast.LENGTH_LONG).show());
```

Comment:
I won't because that has changed already in the code for the next PR:

![image](https://cloud.githubusercontent.com/assets/153802/22864194/c17de080-f144-11e6-9839-77f263ab4dae.png)

Only 1 line changes.